src/main/java/ch/njol/skript/expressions/ExprExperienceChangeReason.java

```diff
@@ -0,0 +1,40 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.EventValueExpression;
+import org.bukkit.event.Event;
+import org.bukkit.event.player.PlayerExpCooldownChangeEvent.ChangeReason;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Experience Change Reason")
```

Comment:
This is specifically about the cooldown's change reason right? Not the experience?